### PR TITLE
docs(interactive): Add document about `column_mapping`

### PR DESCRIPTION
**Pull request by zhanglei1949**
Clarifying the behavior of interactive when importing data from diverse sources.

### CODE REVIEW

File: docs/flex/interactive/data_import.md [hunk 1]
```diff
@@ -6,6 +6,28 @@ In our guide on [using custom graph data](./custom_graph_data.md), we introduced
...
+    index: 0  # Column index in the data source
+    name: col_name # If a column name is present
+  property: property_name  # The mapped property name
+```
+
+The column mapping requirements differ based on the data source:
+
+#### Import from CSV 
+
+You can provide either `index`, `name`, or both.
```
Comment (lidongze0629) [inline docs/flex/interactive/data_import.md:25]: which one has higher priority if `index` and `name` exist simultaneously ?

Comment (zhanglei1949) [reply docs/flex/interactive/data_import.md]: Updated in documentation.
If Both name and index are specified, we will check whether they points to the same column.

**Review by lidongze0629 — APPROVED** [review]
LGTM.

File: docs/flex/interactive/data_import.md (hunk 1 above)
Comment (longbinlai) [reply docs/flex/interactive/data_import.md]: What if they are not?

Comment (zhanglei1949) [reply docs/flex/interactive/data_import.md]: If not, exception will be returned.

**Review by liulx20 — APPROVED** [review]

**Pull request merged**